pyproject.toml

```diff
@@ -0,0 +1,26 @@
+# Configuration for Black
+
+# NOTE: you have to use single-quoted strings in TOML for regular expressions.
+# It's the equivalent of r-strings in Python.  Multiline strings are treated as
+# verbose regular expressions by Black.  Use [ ] to denote a significant space
+# character.
+
+[tool.black]
+line-length = 88
+target_version = ['py35', 'py36', 'py37']
+include = '\.pyi?$'
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | venv
+)/
+'''
```

Comment:
nit: add a line break at the end of a file.

Reply:
thanks, added!
i also notice line length used in koalas is 100, not 88 when reading through `contributing.md`, also changed in this file.